compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Base.hs

```diff
@@ -228,6 +247,8 @@ data BuiltinExpr
   -- Numerical conversion
   | BEInt64ToDecimal           -- :: Int64 -> Decimal, always succeeds since 10^28 > 2^63
   | BEDecimalToInt64           -- :: Decimal -> Int64, only converts the whole part, crashes if it doesn't fit
+  | BEInt64ToNumeric           -- :: ∀(s:nat). Int64 -> Numeric s, crashes if it doesn't fit (TODO: verify?)
```

Comment:
It crashes indeed. This is different from the Decimal version where the result always fit.
you can always double check in the [spec](https://github.com/digital-asset/daml/blob/master/daml-lf/spec/daml-lf-1.rst#conversions-functions)